activesupport/lib/active_support/cache/strategy/local_cache.rb

```diff
@@ -107,6 +107,12 @@ def cleanup(**options) # :nodoc:
           super
         end
 
+        def delete_matched(key, options = nil) # :nodoc:
```

Comment:
I missed this last time, looks like the parameter is called `matcher` everywhere else:

```suggestion
        def delete_matched(matcher, options = nil) # :nodoc:
```